scripts/histmakers/w_z_gen_dists.py

```diff
@@ -153,7 +171,10 @@ def build_graph(df, dataset):
     return results, weightsum
 
 resultdict = narf.build_and_run(datasets, build_graph)
-output_tools.write_analysis_output(resultdict, f"{os.path.basename(__file__).replace('py', 'hdf5')}", args, update_name=not args.forceDefaultName)
+if(args.ptqVgen):
+    output_tools.write_analysis_output(resultdict, f"{os.path.basename(__file__).replace('.py', f'_pdf_{args.pdfs[0]}_vars_qtbyQ.hdf5')}", args, update_name=not args.forceDefaultName)
```

Comment:
Here the name is modified to contain the chosen PDF set when you want to use qtT/Q. Do you really need that information from the name? It should be already available from the metadata, given that it depends on a command option. 
Also, minor comment, I would say "qtOverQ" is easier to understand that "qtbyQ", if the meaning is the ratio.